components/app/dao-project-overview/DaoProjectsHeader.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <LandingSectionContainer>
+    <h2>{{ $t('dao-projects.header.title') }} <span class="text-primary">{{ $t('dao-projects.header.titleHighlight') }}</span></h2>
+    <p>{{ $t('dao-projects.header.text') }}</p>
+    <div class="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 2xl:grid-cols-5 gap-4 mt-10">
+      <div v-for="card in daoInfoCards" :key="card.title">
+      <div class="shadow p-4 pb-4">
+          <div class="flex align-center">
+              <div class="mr-3">
+                <img
+                  class="inline-block mt-3 h-6 w-6 rounded-full ring-2 ring-white"
+                  :src="card.imageURL"
+                  :alt="card.title"
+                />
+              </div>
+              <div>
+                <p class="text-primary small-text text-primary truncate">{{ card.title | truncate(18) }}</p>
```

Comment:
question (non-blocking): do you need the truncate filter here? Seems like you are already planning to use the tailwind class for it?

Reply:
For now yes, but we will replace them after we updated to the newest version.